packages/flutter_locations/lib/src/ui/widgets/map.dart

```diff
@@ -119,7 +149,14 @@ class LocationsMap extends HookWidget {
                   initialZoom: initialZoom,
                   onMapReady: initializeMaps,
                 ),
-                children: layers,
+                children: [
+                  if (config.isFallbackMapType)
+                    TileLayer(
```

Comment:
its not possible to add the poi styling to openstreetmap. This needs to be done differently if these functionalities need to be included.